scripts/start.js

```diff
@@ -48,12 +51,24 @@ let isElectronRunning = false;
  * Prevents multiple re-runs of Electron App
  */
 function runElectronApp() {
-  if (isElectronRunning)
-    return;
+  if (isElectronRunning) return;
 
   isElectronRunning = true;
-
-  exec(`ELECTRON_START_URL=http://localhost:${DEFAULT_PORT} electron .`,
+  // For Windows Support
+  const envPATHS = process.env.PATH.split(';');
+  envPATHS.push(path.join(__dirname, '../node_modules', '.bin'));
+  const newPATHS = envPATHS.join(';');
+
+  const electronCommand = IS_WIN ? 'electron.cmd' : 'electron';
+
+  exec(
+    `${electronCommand} .`,
+    {
+      env: {
+        PATH: IS_WIN ? newPATHS : process.env.PATH,
```

Comment:
@joshwcomeau @bennygenel I'm working on the fix for Ubuntu. My last commit is still not working but I made some progress - not pushed yet.

I have a question to `PATH` on `exec`. Why is it needed? The app will also work with-out it or do I miss where it is needed?
The problem with electron not starting is because `env.ELECTRON_START_URL` is not correctly available to electron. A fix would be to add it to `process.env` like this:

    process.env['ELECTRON_START_URL'] = process.env['ELECTRON_START_URL'] || `http://localhost:${port}`;

So my electron start command would look like this:

    exec(
      `${electronCommand} .`,
      (err, stdout, stderr) => { ... unchanged });

Note: `port` at `process.env` is passed to `function runElectronApp(port){ }` because it's possible that `checkPort` is opening a different port if port 3000 is already in use.